Elastica/Indexor/ContentIndexor.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace OpenOrchestra\Elastica\Indexor;
+
+use Elastica\Client;
+use Elastica\Index;
+use OpenOrchestra\Elastica\Transformer\ModelToElasticaTransformerInterface;
+use OpenOrchestra\ModelInterface\Model\ContentInterface;
+
+/**
+ * Class ContentIndexor
+ */
+class ContentIndexor implements DocumentIndexorInterface, MultipleDocumentIndexorInterface
+{
+    protected $client;
+    protected $transformer;
+
+    /**
+     * @param Client                              $client
+     * @param ModelToElasticaTransformerInterface $transformer
+     */
+    public function __construct(Client $client, ModelToElasticaTransformerInterface $transformer)
+    {
+        $this->client = $client;
+        $this->transformer = $transformer;
+    }
+
+    /**
+     * Index the object after a transformation
+     *
+     * @param ContentInterface $content
+     */
+    public function index($content)
```

Comment:
Yes, it is a contentInterface.

As I have made an interface, I think that I will do as in the transformer and throw an exception if the type is not respected